RELEASE_NOTES.md

```diff
@@ -1,5 +1,115 @@
 # Release Notes
 
+# Staging 2019/08/14
+## Ingest
+### Core v0.9.4.rc
+* Memory-optimized findAssays() for stability
+* Updated primary and secondary submission documentation
+* New API endpoints for linking process to input bundle and input files
+
+### Exporter v0.8.8.rc
+* Exporting major/minor schema versions in provenance
+* Fix null:null submission error
+
+### UI v0.10.1.rc
+* Display project uuid in submission view
+
+## Data Store
+### Version: f9cc3c3eabdf2e2b90943f76c1e3af4225869f0c
```

Comment:
Data Store has a different version in the release notes:
"2019-08-14-16-06-17-staging.release"